build/MSBuildToolset/csc

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+THISDIR=$(dirname $0)
+
+chmod +x $THISDIR/corerun 2>/dev/null
+$THISDIR/corerun $THISDIR/csc.exe "$@"
```

Comment:
No, that file is copied to the output directory. The build outputs to `Binaries/toolset`